### PR TITLE
Latest prototype build (netcore 2.2 !)

### PR DESCRIPTION
**Pull request by Adrriii**
Changes in this PR:
- Heavy arcs handling optimization
- Add autoplay feature (to be enabled in GameplayEngine:17)
- Switch target to Netcore 2.2 for better linux compatibility and overall performance
- Apply the game icon

### CODE REVIEW

**Comment by Adrriii** [comment]
New changes:
- Assets Manager (Need to fill the Unload method later on)
- Remove some test code in Pulsarc.cs
- Display timed out arcs judgement (Miss)

File: Pulsarc/Content/obj/DesktopGL/default.mgcontent [hunk 1]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PipelineBuildEvent xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <SourceFile>C:/Code/Pulsarc/Pulsarc/Content/default.png</SourceFile>
-  <SourceTime>2019-07-22T23:48:42.8758535+02:00</SourceTime>
-  <DestFile>C:/Code/Pulsarc/Pulsarc/Content/bin/DesktopGL/default.xnb</DestFile>
-  <DestTime>2019-07-22T23:49:03.5908386+02:00</DestTime>
+  <SourceFile>C:/Code/ttemp/Pulsarc/Content/default.png</SourceFile>
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/default.mgcontent]: Remove this file

File: Pulsarc/Content/obj/DesktopGL/Fonts/DefaultFont.mgcontent [hunk 2]
```diff
@@ -1,9 +1,9 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PipelineBuildEvent xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <SourceFile>C:/Code/Pulsarc/Pulsarc/Content/Fonts/File.spritefont</SourceFile>
+  <SourceFile>C:/Code/ttemp/Pulsarc/Content/Fonts/DefaultFont.spritefont</SourceFile>
   <SourceTime>2019-07-22T19:54:44.2912914+02:00</SourceTime>
-  <DestFile>C:/Code/Pulsarc/Pulsarc/Content/bin/DesktopGL/Fonts/DefaultFont.xnb</DestFile>
-  <DestTime>2019-07-22T23:49:03.7962274+02:00</DestTime>
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/Fonts/DefaultFont.mgcontent:6]: Remove this file

File: Pulsarc/Content/obj/DesktopGL/Content/default.mgcontent [hunk 3]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PipelineBuildEvent xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
   <SourceFile>C:/Code/Pulsarc/Pulsarc/Content/default.png</SourceFile>
-  <SourceTime>2019-07-22T23:48:42.8758535+02:00</SourceTime>
+  <SourceTime>2019-07-23T14:48:48.8084553+02:00</SourceTime>
   <DestFile>C:/Code/Pulsarc/Pulsarc/Content/bin/DesktopGL/Content/default.xnb</DestFile>
-  <DestTime>2019-07-23T00:14:42.0097011+02:00</DestTime>
+  <DestTime>2019-07-23T15:13:32.8504631+02:00</DestTime>
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/Content/default.mgcontent]: Remove this file

File: Pulsarc/Content/obj/DesktopGL/Content/Fonts/DefaultFont.mgcontent [hunk 4]
```diff
@@ -1,13 +1,13 @@
 <?xml version="1.0" encoding="utf-8"?>
 <PipelineBuildEvent xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance">
-  <SourceFile>C:/Code/Pulsarc/Pulsarc/Content/Fonts/File.spritefont</SourceFile>
-  <SourceTime>2019-07-22T19:54:44.2912914+02:00</SourceTime>
+  <SourceFile>C:/Code/Pulsarc/Pulsarc/Content/Fonts/DefaultFont.spritefont</SourceFile>
+  <SourceTime>2019-07-23T14:48:48.8084553+02:00</SourceTime>
   <DestFile>C:/Code/Pulsarc/Pulsarc/Content/bin/DesktopGL/Content/Fonts/DefaultFont.xnb</DestFile>
-  <DestTime>2019-07-22T19:54:54.941931+02:00</DestTime>
+  <DestTime>2019-07-23T15:13:33.0728675+02:00</DestTime>
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/Content/Fonts/DefaultFont.mgcontent]: Remove this file

File: Pulsarc/Content/obj/DesktopGL/Content/.mgcontent [hunk 5]
```diff
@@ -5,10 +5,6 @@
   <Config />
   <SourceFiles>
     <File>C:/Code/Pulsarc/Pulsarc/Content/default.png</File>
-    <File>C:/Code/Pulsarc/Pulsarc/Content/Fonts/File.spritefont</File>
+    <File>C:/Code/Pulsarc/Pulsarc/Content/Fonts/DefaultFont.spritefont</File>
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/Content/.mgcontent]: Remove this file

File: Pulsarc/Content/obj/DesktopGL/.mgstats [hunk 6]
```diff
@@ -1,3 +1,3 @@
 Source File,Dest File,Processor Type,Content Type,Source File Size,Dest File Size,Build Seconds
-"C:/Code/Pulsarc/Pulsarc/Content/default.png","C:/Code/Pulsarc/Pulsarc/Content/bin/DesktopGL/default.xnb","TextureProcessor","Texture2DContent",1889,10085,0,1765269
-"C:/Code/Pulsarc/Pulsarc/Content/Fonts/File.spritefont","C:/Code/Pulsarc/Pulsarc/Content/bin/DesktopGL/Fonts/DefaultFont.xnb","FontDescriptionProcessor","SpriteFontContent",2068,267281,0,1964508
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/.mgstats:3]: Remove this file

File: Pulsarc/Content/obj/DesktopGL/.mgcontent [hunk 7]
```diff
@@ -4,11 +4,11 @@
   <Platform>DesktopGL</Platform>
   <Config />
   <SourceFiles>
-    <File>C:/Code/Pulsarc/Pulsarc/Content/default.png</File>
-    <File>C:/Code/Pulsarc/Pulsarc/Content/Fonts/File.spritefont</File>
+    <File>C:/Code/ttemp/Pulsarc/Content/default.png</File>
+    <File>C:/Code/ttemp/Pulsarc/Content/Fonts/DefaultFont.spritefont</File>
```
Comment (jaanjah) [inline Pulsarc/Content/obj/DesktopGL/.mgcontent]: Remove this file

**Comment by Adrriii** [comment]
Added the Accuracy meter !

**Review by ghost — APPROVED** [review]

**Pull request merged**